guides/source/action_mailer_basics.md

```diff
@@ -3,155 +3,143 @@
 Action Mailer Basics
 ====================
 
-This guide provides you with all you need to get started in sending
-emails from your application, and many internals of Action
-Mailer. It also covers how to test your mailers.
+This guide covers sending emails from your Rails application.
 
 After reading this guide, you will know:
 
-* How to send email within a Rails application.
-* How to generate and edit an Action Mailer class and mailer view.
+* How to generate and edit Action Mailer classes and mailer views.
+* How to send attachments and multipart emails.
+* How to use Action Mailer callbacks.
 * How to configure Action Mailer for your environment.
-* How to test your Action Mailer classes.
+* How to preview emails and test your Action Mailer classes.
 
 --------------------------------------------------------------------------------
 
 What is Action Mailer?
 ----------------------
 
-Action Mailer allows you to send emails from your application using mailer classes
-and views.
+Action Mailer allows you to send emails from your Rails application. It's one of the two email related components in the Rails framework. Action Mailer, which is for sending emails and [Action Mailbox](action_mailbox_basics.html), which deals with receiving emails.
```

Comment:
```suggestion
Action Mailer allows you to send emails from your Rails application. It's one of two email-related components in the Rails framework: 

1. Action Mailer, for sending emails
2. [Action Mailbox](action_mailbox_basics.html), for receiving emails
```

Reply:
Small suggestion to make the two components ultra clear

Reply:
updated